homeassistant/components/here_travel_time/sensor.py

```diff
@@ -229,6 +224,17 @@ def __init__(
         if self._destination_entity_id is None:
             self._here_data.destination = destination
 
+    async def async_added_to_hass(self) -> None:
+        """Delay the sensor update to avoid entity not found warnings."""
+
+        def delayed_sensor_update(event):
```

Comment:
Please decorate this with `@callback` imported from core.py. That will make sure the callback is run the event loop.